src/index.js

```diff
@@ -90,29 +90,37 @@ function buildArgs(options) {
 // function creates the conf object to be written to that temp file before
 // passing to jsdoc as a command line argument.
 function buildConf(options) {
-    let opts = options || {};
-
+    const opts = options || {};
+    const config = typeof opts.config === 'object' && !Array.isArray(opts.config)
```

Comment:
Checking that `opts.config` is an `object` as well as that it's not an `array` since `typeof  [] === 'array`.

Reply:
I opted for calling the top-level config `config`, but I'm open to a different name. I thought the name fit semantically.

Reply:
In addition if `config` is a thenable (Promise) `typeof` would resolve it to an object. I might make a change to use `_.isObject` instead of all this type check as lodash handles these JavaScript nuances very well.

In fact an update to simply use the lodash type checking methods in lieu of `typeof` might be a good idea.